src/components/OurWork.astro

```diff
@@ -1,209 +1,201 @@
-    }
+	0% {
+		color: var(--tertiary);
+		opacity: 0;
+		transform: translateX(40px);
+	}
 
-    90% {
-        opacity: 1;
-    }
+	90% {
+		opacity: 1;
+	}
 
-    100% {
-        opacity: 1;
-        -webkit-transform: translateX(0px);
-        transform: translateX(0px);
-    }
+	100% {
+		opacity: 1;
+		transform: translateX(0px);
+	}
 }
 
 @keyframes quote-out {
-    0% {
-        opacity: 1;
-        -webkit-transform: translateX(0px);
-        transform: translateX(0px);
-    }
+	0% {
+		opacity: 1;
+		transform: translateX(0px);
+	}
 
-    90% {
-        color: var(--secondary);
-        opacity: 0;
-    }
+	90% {
+		color: var(--secondary);
+		opacity: 0;
+	}
 
-    100% {
-        opacity: 0;
-        -webkit-transform: translateX(-40px);
-        transform: translateX(-40px);
-    }
+	100% {
+		opacity: 0;
+		transform: translateX(-40px);
+	}
 }
 
 h2 {
-    font-size: 30pt;
-}
-#work-title {
-    margin-top: 130px;
-}
-#work-title a {
-    text-align: center;
-    width: 20%;
-    margin: 0 auto;
-    display: block;
-    text-decoration: unset;
-    color: var(--white);
-}
-
-a:hover::after {
-    opacity: 1;
-    width: 100%;
-}
-
-.link::after {
-    content: '';
-    margin-top: 0.3rem;
-    display: block;
-    width: 0;
-    opacity: 0.7;
-    height: 2px;
-    background: var(--white);
-    transition: width 0.3s;
-}
-#our-work {
-    max-width: 900px;
-    width: 95%;
-    height: 300px;
-    margin: 10px auto 18px auto;
-    position: relative;
-    overflow: hidden;
+	font-size: 2.5rem;
+	margin-top: 130px;
+}
+
+h2 a {
+	display: block;
+	text-decoration: none;
+	color: var(--white);
+
+	max-width: 8ch;
+	max-width: max-content;
+	margin: auto;
+	text-align: center;
+}
+
+h2 a::after {
+	content: '';
+	margin-top: 0.3rem;
+	display: block;
+	width: 0;
+	opacity: 0.7;
+	height: 2px;
+	background: var(--white);
+	transition: width 0.3s;
+}
+
+h2 a:hover::after {
+	opacity: 1;
+	width: 100%;
+}
+
+.our-work {
+	max-width: 900px;
+	width: 95%;
+	height: 300px;
+	margin: 10px auto 18px auto;
+	position: relative;
+	overflow: hidden;
 }
+
 @media screen and (max-width: 800px) {
-    #our-work {
-        height: 450px;
-    }
+	.our-work {
+		height: 450px;
+	}
 }
 
 @media screen and (max-width: 620px) {
-    #our-work {
-        height: 500px;
-    }
+	.our-work {
+		height: 500px;
+	}
 }
 @media screen and (max-width: 320px) {
-    #our-work {
-        height: 700px;
-    }
-}
-#button-continer {
-    position: relative;
+	.our-work {
+		height: 700px;
+	}
 }
 
 .work {
-    position: absolute;
-    top: 0;
-    margin-left: auto;
-    margin-right: auto;
-    left: 0;
-    right: 0;
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
 }
 
 .work h3 {
-    text-align: center;
-    font-size: 22pt;
+	text-align: center;
+	font-size: 1.6rem;
 }
 
 .work p {
-    line-height: 27px;
+	line-height: 1.8;
 }
 
-.read-more {
-    display: inline-block;
-    color: var(--secondary);
-    text-decoration: none;
+a.read-more {
```

Comment:
Sometimes I like to over-specify because it reminds me what type of element it is. `a.read-more` reads like _"the 'read more' link"_ to me.